test/contact-forms.spec.js

```diff
@@ -119,17 +119,26 @@ describe('contact forms', () => {
   });
 
   it('#59 - msf-goma create person', async () => {
-    const result = await harness.fillContactForm('goma-person', [
+    const result = await harness.fillContactCreateForm('goma-person', [
       'PARENT', 'patient', '123', 'Full Name', '1990-10-08', 'male',
       '555-123-4567', 'false', 'eng', 'yes', 'second', 'no', 'unknown',
       ['diabetes'], '', 'notes'
     ]);
+    expect(result.errors).to.be.empty;
+  });
 
+  it('create and edit household contact', async () => {
+    let result = await harness.fillContactCreateForm('household_contact', ...[['Head', 'female', 'over5', 'no', '20', '', '', 'sister']]);
+    expect(result.errors).to.be.empty; // create the person
+    const contactBeforeEdit = { ...result.contacts[0] };
```

Comment:
i think you are only shallow copying here.  i suggest that you do a true deep copy the object.  Like `_.cloneDeep()` for example https://www.geeksforgeeks.org/lodash-_-clonedeep-method/